Another meaningless change

Dom-Bach opened the pull request:
Will enable a pull request to the master branch (for peer review in course)

Dom-Bach committed:
Another meaningless change

Will enable a pull request to the master branch

fdabl reviewed (approved):
Your README looks very good (and is quite ambitious!). Two things: First, you get the help entry you mention under "what to add, if possible" for free if you document your functions; and since you are required to document your functions, you will have such help entries. Second, I recommend you give your commits descriptive names; "Another meaningless change" is not descriptive. Here, you could have written "Fix typo" or something along those lines :-)

Dom-Bach committed:
Patch 1.1: PsychoPy experiment + example stimuli

-(skeleton) Code of PsychoPy masking experiment with the most important functions
-Provide example stimuli

Next:
-Make code run properly
-Clean code according to style guide
